plugins/system/userlogs/userlogs.php

```diff
@@ -17,26 +17,54 @@
  */
 class PlgSystemUserLogs extends JPlugin
 {
-    protected $loggable_extensions = array();
+	/**
+	 * Array of loggable extensions.
+	 *
+	 * @var    array
+	 * @since  3.6
+	 */
+	protected $loggable_extensions = array();
 
-    protected $jinput;
+	/**
+	 * Application object.
+	 *
+	 * @var    JApplicationCms
+	 * @since  3.6
+	 */
+	protected $app;
 
-    protected $db;
+	/**
+	 * Database object.
+	 *
+	 * @var    JDatabaseDriver
+	 * @since  3.6
+	 */
+	protected $db;
 
-    public function __construct(&$subject, $config)
+	/**
+	 * Constructor.
+	 *
+	 * @param   object  &$subject  The object to observe.
+	 * @param   array   $config    An optional associative array of configuration settings.
+	 *
+	 * @since   3.6
+	 */
+	public function __construct(&$subject, $config)
 	{
 		parent::__construct($subject, $config);
-        if(is_array($this->params->get('loggable_extensions')))
-        {
-            $this->loggable_extensions = $this->params->get('loggable_extensions');
-        }
-        else
-        {
-            $this->loggable_extensions = explode(',', $this->params->get('loggable_extensions'));
-        }
-        $this->jinput = JFactory::getApplication()->input;
-    }
-    /**
+
+		if (is_array($this->params->get('loggable_extensions')))
+		{
+			$this->loggable_extensions = $this->params->get('loggable_extensions');
+		}
+
+		else
```

Comment:
before else you don't need an empty line (yes i know the code style can be irritating in the beginning).